src/core/dbus-manager.c

```diff
@@ -1451,6 +1451,27 @@ int verify_run_space_and_log(const char *message) {
         return 0;
 }
 
+static void log_reload_caller(sd_bus_message *message, Manager *manager) {
+        _cleanup_(sd_bus_creds_unrefp) sd_bus_creds *creds = NULL;
+        Unit *caller;
+        pid_t pid;
+
+        assert(message);
+        assert(manager);
+
+        if (sd_bus_query_sender_creds(message, SD_BUS_CREDS_PID, &creds) < 0)
+                return;
+
+        if (sd_bus_creds_get_pid(creds, &pid) < 0)
+                return;
+
+        caller = manager_get_unit_by_pid(manager, pid);
+        if (!caller)
+                log_info("Reloading requested by process '"PID_FMT"'...", pid);
+        else
+                log_info("Reloading requested by unit '%s'...", caller->id);
```

Comment:
Shouldn't this also log the PID? "by unit %s (PID "PIDFMT")".

Also please drop `''` from around PID — it's a number and shouldn't be quoted.